CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Changelog
 
-## 2.0.5
+## Unreleased
+
+Fix bug in API docs that shows required attributes as optional (#113)
```

Comment:
Sorry @kevindew, closed #113 because I had misunderstood the relationship between JSON Schema and OpenAPI.